api/src/services/critterbase-service.ts

```diff
@@ -174,7 +204,7 @@ export interface IBulkCreate {
   collections?: ICollection[];
   mortalities?: IMortality[];
   locations?: ILocation[];
-  markings?: IMarking[];
+  markings?: IMarking[] | IBulkCreateMarking[];
```

Comment:
Just checking: This `IBulkCreate` interface is used by both the `updateCritter` and `bulkCreate` functions in `critterbase-service.ts`. Are they both okay with receiving either `IMarking[]` or `IBulkCreateMarking[]`?

Reply:
I wrote a TODO at the top of this file kinda referencing this. It looks like a lot of the types in this file are being used as request bodies AND response bodies which is pretty messy. 

I think we should go through and audit this service / clean up some of the strange types.

I think also simplifying the strange constant mapping for endpionts and use a better query param parser.

Reply:
The ```updateCritter``` method is actually just hitting the bulkUpdate endpoint